spec/acceptance/apt_backports_spec.rb

```diff
@@ -3,6 +3,21 @@
 require 'spec_helper_acceptance'
 
 describe 'apt::backports' do
+  context 'update_had_no_effect script' do
+    let(:pp) do
+      <<-MANIFEST
+        exec { 'Test the script':
+          command   => epp('apt/update_had_no_effect.sh.epp'),
```

Comment:
Since this is code in the shell language, I think we need to set `provider => 'shell'` for this test to pass.

Reply:
Done.